rrfs-test/CMakeLists.txt

```diff
@@ -47,5 +76,11 @@ ecbuild_add_test( TARGET   ${target_test}
                   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/rundir-${target_test}
                   ARGS     ${target_test}.yaml 
                   COMMAND  fv3jedi_var.x )
+set(target_test rrfs_mpasjedi_2022052619_Ens3Dvar)
```

Comment:
the build script has the option for either or both dycore right? should we add CMake to only build these tests if that dycore is built?

Reply:
@CoryMartin-NOAA  Good suggestion. Will add this condition for them.